doc/nrf/external_comp/nrf_cloud.rst

```diff
@@ -34,6 +34,46 @@ You can use the services offered by nRF Cloud in the following scenarios:
 * Device connected to nRF Cloud over REST, interacting using the `nRF Cloud REST API`_.
 * Device connected to a customer cloud service in a suitable manner. The services can be used from the customer cloud service that communicates over REST to the nRF Cloud REST API in a proxy configuration.
 
+Choosing a device ID format: prefix-IMEI, device UUID, or other
+***************************************************************
+
+nRF Cloud supports a variety of device ID formats:
```

Comment:
the `nrf_cloud` firmware library supports...
nRF Cloud supports pretty much anything 😆